heartbeat/heartbeat.full.yml

```diff
@@ -189,6 +189,9 @@ heartbeat.monitors:
     # Required response contents.
     #body:
 
+    # Required response contents must contain value.
+    #body_contains:
```

Comment:
This file is generated. You need to modify in `heartbeat/_meta/beat.full.yml` and then run `make update`.